lern-data-products/src/main/scala/org/sunbird/core/exhaust/BaseReportsJob.scala

```diff
@@ -54,6 +54,9 @@ trait BaseReportsJob {
       case "s3" =>
         spark.sparkContext.hadoopConfiguration.set("fs.s3n.awsAccessKeyId", AppConf.getConfig(storageKey));
         spark.sparkContext.hadoopConfiguration.set("fs.s3n.awsSecretAccessKey", AppConf.getConfig(storageSecret));
+      case "oci" =>
+        spark.sparkContext.hadoopConfiguration.set("fs.s3n.awsAccessKeyId", AppConf.getConfig(storageKey));
+        spark.sparkContext.hadoopConfiguration.set("fs.s3n.awsSecretAccessKey", AppConf.getConfig(storageSecret));
```

Comment:
Code is same for s3 and oci. Please use the pipe in case to match multiple values. 
Sample code:
`code match {
  case "1" | "2" => // whatever
  case "3" =>
}`

Reply:
made changes as suggested